main.ts

```diff
@@ -38,24 +31,14 @@ const {
   use,
   visit,
 } = fork([
-  // This core module of AST types captures ES5 as it is parsed today by
-  // git://github.com/ariya/esprima.git#master.
-  coreDef,
-
   // Feel free to add to or remove from this list of extension modules to
   // configure the precise type hierarchy that you need.
-  es6Def,
-  es2016Def,
-  es2017Def,
-  es2018Def,
-  es2019Def,
-  es2020Def,
+  esProposalsDef,
```

Comment:
Since `esProposalsDef` imports all previous specification years and the core definitions, we can/should use only `esProposalsDef` here, so types will not be prematurely finalized by loading previous years first. Note that removing the previous defs does cause some harmless reordering of generated types.